ouroboros-consensus-diffusion/app/conformance-test-runner/Main.hs

```diff
@@ -32,24 +28,33 @@ import Ouroboros.Network.PeerSelection (PeerAdvertise (..), PortNumber)
 import Ouroboros.Network.PeerSelection.LedgerPeers (RelayAccessPoint (..), UseLedgerPeers (..))
 import Ouroboros.Network.PeerSelection.State.LocalRootPeers (HotValency (..), WarmValency (..))
 import Server (run)
-import System.Exit (ExitCode (..))
+import System.Environment (getArgs)
+import System.Exit (ExitCode (..), exitSuccess, exitWith)
+import System.IO (hPutStrLn, stderr)
 import Test.Consensus.PointSchedule (PointSchedule (..))
 import Test.Consensus.PointSchedule.Peers (PeerId (..), Peers (Peers), getPeerIds)
 
+-- | Exit statuses for the test runner. 'TestPassed' is represented
+-- by an empty set of 'StatusFlags'.
 data ExitStatus = InternalError | BadUsage | Flags (Set StatusFlag)
 
-pattern Success :: ExitStatus
-pattern Success <- Flags (null -> True)
+pattern TestPassed :: ExitStatus
```

Comment:
I liked `Success` better, personally --- since it's more inclusive and is a correct exit code for things like `--help`. Why the change?